src/wiki/plugins/macros/mdx/macro.py

```diff
@@ -35,37 +35,37 @@ def extendMarkdown(self, md):
         md.postprocessors.add('unescaper', SubstitutionPostprocessor(), "_begin")
 
 
-class SubstitutionPreprocessor(markdown.preprocessors.Preprocessor):
-    def __init__(self):
-        self.subs = {
-            "`": "👻🖤BACKTICK🖤👻"
-        }
+# Escaping --------------------------------------------------------------------------------
+ESCAPED = ["`"]
```

Comment:
@nschneid if there are any other characters you want to escape within macros (to avoid having them interpreted as markdown) we can add them here